lib/providers/gcm.js

```diff
@@ -70,6 +70,7 @@ GcmProvider.prototype._createMessage = function(notification) {
   Object.keys(notification).forEach(function (key) {
     if (notification[key] !== null && typeof notification[key] !== 'undefined') {
       message.addData(key, notification[key]);
+      message.addNotification(key, notification[key]);
```

Comment:
I was wondering if we should allow the notification argument to have separate data and notification properties, for example:
```js
{
  x: 1,
  notifications: {
    y: 2
  }
}
```